### DISCUSSION TITLE
Comparison to Greenlight v3

### DISCUSSION THREAD

**Discussion by s-kucharski** [THM-Health/PILOS#567] [Ideas]
Thank you for the detailed [comparison with Greenlight v2](https://github.com/THM-Health/PILOS/wiki/Comparison-Greenlight-and-PILOS). Is there any comparison with Greenlight v3 or a chance that such will be created?

**Comment by samuelwei** [comment]
I'm sorry for not replying, we just missed your question.
If you are using Greenlight v3 and would be willing to contribute such a comparision, feel free to open a PR for this file: https://github.com/THM-Health/PILOS/blob/develop/docs/docs/administration/06-greenlight.md
